canonical/ubuntu.com
# Build new Embeddings 2024 FAQs page

## QA

- Go to /embedding scroll down to Embedding support section and compare with [requested changes in copy doc](https://docs.google.com/document/d/1evAFIXbNztOd5yZee-uV8_OM-s6o27P46yXFaaeEhy0/edit)
- Click on the "Check our FAQs" link and compare with [copy doc](https://docs.google.com/document/d/1xWrM0D4LkWdJh9EiAUuXgbYKqh15BnNcQpXqTlZCuYA/edit)


## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
